nomad/structs/node.go

```diff
@@ -95,6 +95,15 @@ type CSINodeInfo struct {
 	// RequiresNodeStageVolume indicates whether the client should Stage/Unstage
 	// volumes on this node.
 	RequiresNodeStageVolume bool
```

Comment:
In retrospect this totally should have been named `SupportsStageVolume` but it's now persisted in someone's state store this way. 😦 Technically it _is_ required and not merely supported, whereas these other fields are "supported but not required", so it's not so bad... just awkward and ugly.